lib/servers/Pulser2/pulse_sequences/pulse_sequence.py

```diff
@@ -12,7 +12,7 @@ class pulse_sequence(object):
 	required_subsequences = []
 	replaced_parameters = {}
 	
-	def __init__(self, parameter_dict, start = WithUnit(0, 's')):
+	def __init__(self, parameter_dict, start = WithUnit(1.0, 'us')):
```

Comment:
@jayich There was a problem where you could not start pulse sequences at t=0. I am going to look further into this before I merge today